tests/integration/test_util.py

```diff
@@ -17,8 +18,9 @@
     ],
 )
 def test_random_choice_true_mask2d_find_right_coordinates(fixture_slide, binary_mask):
+    np.random.seed(0)
```

Comment:
why is this seed needed?

Reply:
since we are testing a function with a random behaviour it's better for the test to be predictable